Added a method to upload multiple files in sequential as well as...

rprit opened the pull request:
…parallel manner

rprit committed:
Added a method to upload multiple files in sequential as well as parallel manner

sunilraok commented on Program.cs:
```diff
@@ -2,39 +2,155 @@
     {
-        internal static async Task Main(string[] args)
+        internal static IList<string> files = new List<string>();
+        internal const string jsonPath = "F:\\Prithvi\\Projects\\FileUploadToDriveFiles\\Credentials.json";
```
Suggestion: Remove folder paths and take them as inputs.

rprit replied:
I have made the changes. Can you please review?

sunilraok commented on Program.cs:
```diff
@@ -2,39 +2,155 @@
+                    MimeType = GetMimeType(file)
+                };
+
+                FilesResource.CreateMediaUpload request;
```
This can be done inline with the request creation in line 73, right?

rprit committed:
Updates based on review comments

Simplified CreateMediaUpload request. Removed hardcoded folder paths and used user input instead.

rprit committed:
Changed method names
